modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/framework/TestBuilders.java

```diff
@@ -937,8 +937,9 @@ public IgniteTable build() {
                             false,
                             columns.size(),
                             NativeTypes.INT32,
-                            DefaultValueStrategy.DEFAULT_NULL,
-                            null
+                            DefaultValueStrategy.DEFAULT_COMPUTED,
+                            () -> {
+                                throw new AssertionError("Implicit primary key is generated by a function"); }
```

Comment:
misleading message, this column represents partition id, not primary key

Reply:
fixed